src/diffusers/models/controlnet.py

```diff
@@ -774,7 +777,7 @@ def forward(
         sample = self.conv_in(sample)
 
         controlnet_cond = self.controlnet_cond_embedding(controlnet_cond)
-        sample = sample + controlnet_cond
+        sample += controlnet_cond
```

Comment:
Not a type hint/docs change. Maybe we can open a separate PR for clean up ops like this?

Reply:
Sure! Will undo such changes